nevergrad/benchmark/optgroups.py

```diff
@@ -253,6 +253,7 @@ def structured_moo() -> tp.Sequence[Optim]:
         "RecES",
         "RecMixES",
         "RecMutDE",
+        "PymooNSGA2",
```

Comment:
current version is only sequential, #1228 will make it usable in batch mode (but not steady mode)
Maybe be useful to filter out inconsistent settings anyway in the xp